src/components/FindWallet/WalletTable/index.tsx

```diff
@@ -308,27 +323,30 @@ const WalletTable = ({
                       w={{ base: "100%", md: "auto" }}
                       ps={{ base: 0, md: 1.5 }}
                     >
-                      {/* Wallet image */}
-                      <Box
-                        w={{ base: "24px", md: "56px" }}
-                        //
-                      >
-                        <Image
-                          src={wallet.image}
-                          alt=""
-                          objectFit="contain"
-                          boxSize="auto"
-                        />
-                      </Box>
-
                       <Box w={{ base: "100%", md: "auto" }}>
-                        <Stack>
+                        <Flex alignItems="start">
+                          {/* Wallet image */}
+                          <Stack
+                            w={{ base: "24px", md: "56px" }}
+                            h={{ base: "24px", md: "56px" }}
+                            me={4}
+                          >
+                            <Image
+                              src={wallet.image}
+                              alt=""
```

Comment:
Would recommend limiting the size of these images. With the current config, we would be fetching an unnecessarily larger image, like the "block wallet" image that is 500pxs, and then shrinking it with css.

```suggestion
                            <Image
                              src={wallet.image}
                              height={56}
                              alt=""
```

Reply:
@pettinarip height is already defined below, `h={{ base: "24px", md: "56px" }}`

Reply:
`h` is not the same as `height`. `h` is a Chakra shorthand for the css prop. `height` is a prop that is used by the [NextJS image component](https://nextjs.org/docs/app/api-reference/components/image#height) to determine the size of the image served.

Reply:
Not a blocker though but it is recommended.

Reply:
Right, I've already fixed it, thanks